src/components/Parallax.tsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'preact/hooks'
+import ChildrenProp from 'models/ChildrenProp'
+
+export default function ({
+  maxYPixelsDisplacedFromOrigin,
+  children,
+}: ChildrenProp & { maxYPixelsDisplacedFromOrigin: number }) {
+  const [offsetY, setOffsetY] = useState(0)
+  const handleScroll = () => setOffsetY(window.pageYOffset)
```

Comment:
Should we use `useCallback` here or is it not necessary?

Reply:
Don't think it's necessary here